webapp/src/js/services/rules-engine.js

```diff
@@ -54,18 +58,49 @@ angular.module('inboxServices').factory('RulesEngine', function(
 
                   ensureTaskFreshness = Debounce(self.fetchTaskDocsForAllContacts, ENSURE_FRESHNESS_SECS * 1000);
                   ensureTaskFreshness();
+                  ensureTaskFreshnessTelemetryData =
```

Comment:
i'm thinking a bit more about how this data set would be used. i suspect the most valuable metric is  "% cancelation rate for a user" -- is there a denominator available for this already? if not, are you open to adding one?

Reply:
To be honest, I'm not entirely sure why this metric is important to you. 
What would a % cancellation rate say about its impact? Are you worried it runs for users that barely every use tasks and targets? (Those users might as well have tasks and targets directly disabled if performance is a concern).

Reply:
Also, if you're looking for a denominator, we're already tracking every time "initilize" runs, so that can be used as a denominator already.

Reply:
basically i'm not sure if this feature was worth building. I have no idea if it will run or if its useful, or if it interers with a user's session. imo it's easy to forget about features like this that sometimes do stuff in the background. Telemetry is basically all we have to understand its usefulness and impact. I think if its cancelation rate is high, we should consider removing it.

initialize makes sense as a denominator. thanks